### ISSUE TITLE
Env-Var expansion for config

### ISSUE THREAD

**Issue by matthiasbeyer** [matthiasbeyer/imag#792]
We propably want to be able to resolve environment variables from the configuration file, as proposed in #775 

This is not only a nice-to-have but will result in better security at some point.

**Comment by matthiasbeyer** [comment]
@TheNeikos wanna tackle this one? Could be a nice playground for string-replace in parallel on a large TOML tree...

**Comment by matthiasbeyer** [comment]
This could be marked as WIP as we currently do variable expansion in config stuff in #1023 via handlebars.

**Comment by matthiasbeyer** [comment]
I guess this is rather easy to do, simply by pushing all environment variables into handlebars.

**Comment by matthiasbeyer** [comment]
Won't do this as I don't see the necessity anymore.
